README.md

```diff
@@ -1,7 +1,138 @@
+
+<div align="center">
+  <h3 align="center">
+
+[Primer's](https://primer.io) Official Universal Checkout iOS SDK
+
+  </h3>
+</div>
+
+<br/>
+
+<div align="center"><img src="./assets/checkout-banner.gif?raw=true"  width="50%"/></div>
+
+<br/>
+<br/>
+
+# 💪 Features of the iOS SDK
+
+<p>💳 &nbsp; Create great payment experiences with our highly customizable Universal Checkout</p>
+<p>🧩 &nbsp; Connect and configure any new payment method without a single line of code</p>
+<p>✅ &nbsp; Dynamically handle 3DS 2.0 across processors and be SCA ready</p>
+<p>♻️ &nbsp; Store payment methods for recurring and repeat payments</p>
+<p>🔒 &nbsp; Always PCI compliant without redirecting customers</p>
+
+
+# 📚 Documentation
+
+Consider looking at the following resources:
+
+- [Documentation](https://primer.io/docs)
+- [Client session creation](https://primer.io/docs/accept-payments/manage-client-sessions/#create-a-client-session)
+- [API reference](https://apiref.primer.io/docs)
+- [Changelogs](https://primer.io/docs/changelog/sdk-changelog/ios)
+- [Detailed iOS Documentation](https://www.notion.so/primerapi/iOS-SDK-ebbf44a733624d17bfd0c3a746f171a2)
+
+
+# 💡 Support
+
+For any support or integration related queries, feel free to [Contact Us](mailto:https://support@primer.io).
+
+
+## 🚀 Quick start
+
+Take a look at our [Quick Start Guide](https://primer.io/docs/get-started/ios) for accepting your first payment with Universal Checkout.
+
+<br/>
+
+# 🧱 Installation
+
+## With CocoaPods
+
+The iOS SDK is available with Cocoapods. Just add the PrimerSDK pod and run `pod install`.
+
+```swift{:copy}
+target 'MyApp' do
+  # Other pods...
+ 
+  # Add this to your Podfile
+  pod 'PrimerSDK' # Add this line
+end
+
+```
+For specific versions of the SDK, please refer to the changelog.
+
+## With Swift Package Manager
+
+The Swift Package Manager is a tool for automating the distribution of Swift code and is integrated into Xcode. In order to add PrimerSDK with Swift Package Manager;
+
+1. Select your project, and then navigate to Package Dependencies
+2. Click on the + button at the bottom-left of the Packages section
+3. Paste https://github.com/primer-io/primer-sdk-ios.git into the Search Bar
+4. Press Add Package
+5. Let Xcode download the package and set everything up
+
+
+<img src="./assets/spm-3.png" />
+
+<br/>
+
+# 👩‍💻 Usage
+
+## 📋 Prerequisites
+
+- 🔑 Generate a client token by [creating a client session](https://primer.io/docs/accept-payments/manage-client-sessions) in your backend.
+- 🎉 _That's it!_
+
+## 🔍 &nbsp;Initializing the SDK
+
+Import the Primer SDK and set its delegate as shown in the following example:
+
+```swift{:copy}
+import PrimerSDK
+ 
+class MyViewController: UIViewController {
+ 
+    override func viewDidLoad() {
+        super.viewDidLoad()
+ 
+        // Initialize the SDK with the default settings.
+        let settings = PrimerSettings()
+        Primer.shared.configure(settings: settings, delegate: self)
```

Comment:
To configure Primer with default `PrimerSettings` we can omit the argument passed to the configuration function.
If the intention was to show the PrimerSetting object, fair enough 👍.
Otherwise, we can apply the suggestion below:
```suggestion
        Primer.shared.configure(delegate: self)
```